packages/sdk-demo/src/components/Layout/index.tsx

```diff
@@ -73,12 +75,21 @@ export const DefaultLayout = ({ children, siderProps }: DefaultLayoutProps) => {
           <Menu />
           <Box
             sx={{
+              width: '100%',
               position: 'absolute',
               bottom: 0,
-              mb: 2,
+              p: 2,
             }}
           >
-            {siderProps?.footer}
+            <Stack direction="column" spacing={2} ml={2}>
+              {/*Themes are unfinished.*/}
+              {/*We want to show the theme switcher only in development mode and on the dev deployment only.*/}
+              {(process.env.NODE_ENV === 'development' ||
+                location.pathname.indexOf('dev.monite.') > 0) && (
```

Comment:
I'd suggest moving the condition into a variable with a name, otherwise, it might be a little bit hard to understand what we're checking here.

[As an example of how it's done for Sentry checks](https://github.com/team-monite/monite-sdk/blob/2504752c8b6ea7c999e0a3fa8754cb74e3df49e9/packages/sdk-react/src/core/services/Sentry.factory.ts#L38)

Reply:
fixed